javascripts/editmode.js

```diff
@@ -77,12 +78,15 @@
         bgPickerContentLightnessClass(bgPickerArea, bgPicker.combinedLightness);
       }
 
+      if (data.colorData) {
+        colorRgb = data.colorData.r + ',' + data.colorData.g + ',' + data.colorData.b;
```

Comment:
At first I tried to avoid using ES6 features to make this template as compatible as possible, but since other parts of the site would probably break on older browsers regardless, we allowed ES6 aswell. Good point, template literals would be good here.